### PR TITLE
EOS-18332: Do not install 3rd party packages in mini provisioner

### PR DESCRIPTION
**Pull request by mariyappanp**

### CODE REVIEW

File: low-level/framework/base/sspl_constants.py [hunk 1]
```diff
@@ -252,6 +252,40 @@
 
 SSPL_CONFIGS = ['log_level', 'cli_type', 'sspl_log_file_path', 'cluster_id', 'storage_enclosure', 'setup', 'operating_system']
 
+# SSPL python 3rd party package dependencies
+pip3s_packages_main = {
```
Comment (cdeshmukh) [inline low-level/framework/base/sspl_constants.py:256]: Can we rename to pip3_3ps_pkgs_main

File: low-level/framework/base/sspl_constants.py [hunk 2]
```diff
@@ -252,6 +252,40 @@
...
+    "python-daemon": "2.2.4",
+    "requests": "2.25.1",
+    "zope.component": "4.6.2",
+    "zope.event": "4.5.0",
+    "zope.interface": "5.2.0",
+    "pyinotify": "0.9.6",
+    "cryptography": "2.3"
+}
+
+pip3s_packages_test = {
```
Comment (cdeshmukh) [inline low-level/framework/base/sspl_constants.py:268]: rename to pip3_3ps_pkgs_test plz

File: low-level/framework/base/sspl_constants.py [hunk 3]
```diff
@@ -252,6 +252,40 @@
...
+    "Flask": "1.1.1"
+}
+
+# SSPL 3rd party RPM dependencies
+rpm_dependencies = [
```
Comment (cdeshmukh) [inline low-level/framework/base/sspl_constants.py:273]: Should have structure to maintain versions too, similar to python pkgs

File: low-level/framework/base/sspl_constants.py (hunk 1 above)
Comment (mariyappanp) [reply low-level/framework/base/sspl_constants.py:256]: Sure. May I know what does `3ps` mean to?

File: low-level/files/opt/seagate/sspl/setup/sspl_post_install.py [hunk 4]
```diff
@@ -79,6 +80,47 @@ def dump_global_config(self):
...
+            "jsonschema": "3.2.0",
+            "pyinotify": "0.9.6",
+            "python-daemon": "2.2.4",
+            "requests": "2.25.1",
+            "zope.component": "4.6.2",
+            "zope.event": "4.5.0",
+            "zope.interface": "5.2.0"
+        }
+        # SSPL 3rd party RPM dependencies
+        rpm_3ps_packages = {
```
Comment (cdeshmukh) [inline low-level/files/opt/seagate/sspl/setup/sspl_post_install.py:101]: these rpm versions also taken into consideration by RE ?

**Review by cdeshmukh — APPROVED** [review]

File: low-level/framework/base/sspl_constants.py (hunk 2 above)
Comment (mariyappanp) [reply low-level/framework/base/sspl_constants.py]: Done.

File: low-level/framework/base/sspl_constants.py (hunk 1 above)
Comment (mariyappanp) [reply low-level/framework/base/sspl_constants.py]: Done.

File: low-level/framework/base/sspl_constants.py (hunk 3 above)
Comment (mariyappanp) [reply low-level/framework/base/sspl_constants.py]: Now, added the version for each rpms.

File: low-level/files/opt/seagate/sspl/setup/sspl_post_install.py (hunk 4 above)
Comment (mariyappanp) [reply low-level/files/opt/seagate/sspl/setup/sspl_post_install.py:101]: Yes, Shailesh agreed.
https://github.com/Seagate/cortx-re/pull/275 has version mapped for requested packages.

**Review by thavanathan — APPROVED** [review]

**Review by cdeshmukh — APPROVED** [review]

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**